### ISSUE TITLE
Deluge seems to crash in some way

### ISSUE THREAD

**Issue by etique57** [binhex/arch-delugevpn#116]
Hello,

With the latest release (08th of July, created by watchtower), it seems Deluge stops working after a few seconds.

In the container logs, I get this:
`2019-07-11 21:40:30,288 DEBG 'watchdog-script' stdout output:
[info] Privoxy process listening on port 8118

2019-07-11 21:40:50,690 DEBG 'watchdog-script' stderr output:
Unhandled error in Deferred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/twisted/internet/base.py", line 1272, in run
    self.mainLoop()
  File "/usr/lib/python3.7/site-packages/twisted/internet/base.py", line 1281, in mainLoop
    self.runUntilCurrent()
  File "/usr/lib/python3.7/site-packages/twisted/internet/base.py", line 902, in runUntilCurrent
    call.func(*call.args, **call.kw)
  File "/usr/lib/python3.7/site-packages/twisted/internet/task.py", line 239, in __call__
    d = defer.maybeDeferred(self.f, *self.a, **self.kw)
--- <exception caught here> ---
  File "/usr/lib/python3.7/site-packages/twisted/internet/defer.py", line 151, in maybeDeferred
    result = f(*args, **kw)
  File "/usr/lib/python3.7/site-packages/deluge/core/alertmanager.py", line 70, in update
    self.handle_alerts()
  File "/usr/lib/python3.7/site-packages/deluge/core/alertmanager.py", line 140, in handle_alerts
    for attr in dir(alert)
  File "/usr/lib/python3.7/site-packages/deluge/core/alertmanager.py", line 141, in <dictcomp>
    if not attr.startswith('__')
builtins.UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 25: invalid continuation byte`

I'm not sure how to start debugging? :)

**Comment by binhex** [comment]
Thats a deluge bug, your best course of action is to post this on the deluge forum as a bug.

**Comment by etique57** [comment]
Thanks, went back to 2.0.3-1-03 and everything's fine.

**Comment by matthewaveryusa** [comment]
@etique57 the downgrade is actually not what fixed it -- you just ended up with clients that validly utf8-encode their name. 

@binhex this is a pretty nasty bug as one peer can stop deluge from running. There's upstream patch for this https://dev.deluge-torrent.org/changeset/4b29436cd5e

I tested this by applying the patch manually to this image's release and I was able to get things running again.
